amgut/lib/data_access/sql_connection.py

```diff
@@ -1,72 +1,204 @@
-from __future__ import division
-
 # -----------------------------------------------------------------------------
-# Copyright (c) 2014--, The American Gut Development Team.
+# Copyright (c) 2014--, The Qiita Development Team.
```

Comment:
I see, it would make sense to mention this in the licensing file.

On (Oct-02-15|19:26), Joshua Shorenstein wrote:
> >  # -----------------------------------------------------------------------------
> > -# Copyright (c) 2014--, The American Gut Development Team.
> > +# Copyright (c) 2014--, The Qiita Development Team.
> 
> I don't think this will get modified, just updated from qiita if necessary. I'll add info to contributing.md about it if wanted.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/american-gut-web/pull/519/files#r41082779